### PR TITLE
404 style updates and bug fix

### PR DESCRIPTION
**Pull request by sbsmith86**
Ok, more updates to 404 page after talks with @lkpttn and @mikefantini.

@aaronschachter can you take a look?

### CODE REVIEW

File: lib/modules/dosomething/dosomething_notfound/dosomething_notfound.module [hunk 1]
```diff
@@ -121,30 +121,31 @@ function dosomething_notfound_preprocess_node(&$vars)  {
...
-      $search_form = drupal_build_form('search_form', $form_state);
+      ),
+    );
+    $search_form = drupal_build_form('search_form', $form_state);
+    // Make the search form availabe to the page.
+    $vars['search_form'] = drupal_render($search_form);
 
-      // Make the search form availabe to the page.
-      $vars['search_form'] = drupal_render($search_form);
+    if (count($vars['raw_search_results']) != 0) {
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_notfound/dosomething_notfound.module:137]: What's happening here?  If there are search results, set the copy and exit?  Comments might be helpful to better explain the logic.

File: lib/themes/dosomething/paraneue_dosomething/templates/notfound/node--notfound.tpl.php [hunk 2]
```diff
@@ -23,21 +23,18 @@
   </header>
 
   <?php if ($disable_results == FALSE): ?>
-    <div class="container notfound-search-results <?php if (!empty($notfound_page_type)) { print $notfound_page_type; } ?>">
+    <div class="container notfound-search-results">
       <div class="wrapper notfound-results">
-        <?php if (isset($results_copy)): ?>
-          <h3 class="search-results-header">
+        <?php if (isset($results_copy) && !empty($notfound_page_type)): ?>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/notfound/node--notfound.tpl.php]: This is pretty confusing too, as far as readability... whats a `$notfound_page_type` ? Whether or not its path-based?

**Comment by sbsmith86** [comment]
@aaronschachter added comments and addressed readability issues in the latest commit. Please review.

**Comment by aaronschachter** [comment]
Thanks @sbsmith86 ! Lookin way more readable. :+1: SHIP IT

**Pull request merged**